h2o-assembly/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
   if (project.hasProperty("doIncludeOrc") && project.doIncludeOrc == "true") {
     compile project(":h2o-orc-parser")
   }
+  compile project(":h2o-parquet-parser")
```

Comment:
It is probably ok for now, but later we need to verify what dependencies parquet needs to be run as part of standalone `h2o.jar`.